bin/start_dev

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+[[ -z $VIRTUAL_ENV ]] && \
+  virtualenv env && \
+  source env/bin/activate
+
+python main.py
```

Comment:
doing "exec python main.py" would ensure that failures to start this will propagate exit status out to the caller